packages/web-api/src/types/request/manifest.ts

```diff
@@ -0,0 +1,547 @@
+export interface Manifest extends Record<string, unknown> {
+  /** @description A group of settings that describe the manifest. */
+  _metadata?: ManifestMetadata; // eslint-disable-line @typescript-eslint/naming-convention
+  /**
+   * @description A group of settings that describe parts of an app's appearance within Slack. If you're distributing
+   * the app via the App Directory, read our {@link https://api.slack.com/start/distributing/guidelines#listing listing guidelines} to pick the best values for these settings.
+   */
+  display_information: ManifestDisplayInformation;
+  /** @description A group of settings corresponding to the Features section of the app config pages. */
+  features?: ManifestFeatures;
+  /** @description A group of settings describing OAuth configuration for the app. */
+  oauth_config?: ManifestOAuthConfig;
+  /** @description A group of settings corresponding to the Settings section of the app config pages. */
+  settings?: ManifestSettings;
+  /** @description A group of settings corresponding to Custom Functions bundled in your application. */
+  functions?: Record<string, ManifestFunction>;
+  // TODO: future additions: external_auth_providers, workflows
+}
+
+interface ManifestMetadata {
+  /**
+   * @description An integer that specifies the major version of the manifest schema to target.
+   * Only `1` and `2` are supported. Defaults to `1`.
+   */
+  major_version?: number;
+  /** @description An integer that specifies the minor version of the manifest schema to target. */
+  minor_version?: number;
+}
+
+interface ManifestDisplayInformation {
+  /** @description A string of the name of the app. Maximum length is 35 characters. */
+  name: string;
+  /**
+   * @description A string with a short description of the app for display to users. Maximum length is 140 characters.
+   */
+  description?: string;
+  /** @description A string with a longer version of the description of the app. Maximum length is 4000 characters. */
+  long_description?: string;
+  /**
+   * @description A string containing a hex color value (including the hex sign) that specifies the background color
+   * used on hovercards that display information about your app. Can be 3-digit (`#000`) or 6-digit (`#000000`) hex
+   * values. Once an app has set a background color value, it cannot be removed, only updated.
+   */
+  background_color?: string;
+}
+
+interface ManifestFeatures {
+  /**
+   * @description A subgroup of settings that describe {@link https://api.slack.com/surfaces/app-home App Home} configuration.
+   * @see {@link https://api.slack.com/surfaces/app-home App Home}.
+   */
+  app_home?: ManifestAppHome;
+  /**
+   * @description A subgroup of settings that describe {@link https://api.slack.com/legacy/enabling-bot-users bot user} configuration.
+   * @see {@link https://api.slack.com/legacy/enabling-bot-users Legacy bots}.
```

Comment:
I don't like linking this "legacy" document here but indeed there is no other document for this

Reply:
I took this from the Manifest fields documentation here: https://api.slack.com/reference/manifests#features

The link originally links to api.slack.com/bot-user which redirects to the above link.